playground/src/pages/InstructionTable.tsx

```diff
@@ -0,0 +1,109 @@
+import { Fragment } from 'react';
+import ReactMarkdown from 'react-markdown';
+import gfm from 'remark-gfm';
+import math from 'remark-math';
+import katex from 'rehype-katex'
+import { assemblerInstructions } from '../data/instructions';
+
+import 'katex/dist/katex.min.css'
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ')
+}
+
+/** TODO: 
+ * Search button does not work yet 
+ * Expand button does not work yet - either remove or implement
+ * Decide on a table layout and pagination
+ * Table loads slowly - consider loading only the first 10 rows and then loading more as the user scrolls 
+ * or use react table (https://react-table.tanstack.com/docs/overview)`
+*/
+export default function InstructionTable() {
+  return (
+    <div className="px-4 sm:px-6 lg:px-8">
+      <div className="sm:flex sm:items-center">
+        <div className="sm:flex-auto">
+          <h1 className="text-xl font-semibold text-gray-900">Instructions</h1>
+          <p className="mt-2 text-sm text-gray-700">
+            A list of all Miden assembly instructions.
+          </p>
+        </div>
+        <div className="mt-4 sm:mt-0 sm:ml-16 sm:flex-none">
+        
+          <button
+            type="button"
+            className="inline-flex items-center justify-center rounded-md border border-transparent bg-indigo-600 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2 sm:w-auto"
+          >
+            Search
```

Comment:
The search button doesn't seem to be working, remove it for now?

Reply:
Indeed there is a comment about it in the code above the function. I will add the search button, when the designer proposes a new design. Issue here https://github.com/0xPolygonMiden/examples/issues/59